packages/suite-analytics/src/constants/index.ts

```diff
@@ -0,0 +1,7 @@
+export const SUITE_ANALYTICS_VERSION = '1.18';
```

Comment:
please remove this whole folder and just create file called `constants.ts` we really need to end index madness